### DISCUSSION TITLE
riproduzione episodi dal sito

### DISCUSSION THREAD

**Discussion by DGuidi** [pdonadeo/e-uscito-joypad#43] [Ideas]
inannzitutto complimenti grossissimi per il lavoro, sono capitato per sbaglio oggi dopo qualche mese ed ho visto gozilioni di nuove features! 
mi chiedevo: visto che il play di fatto rimanda al sito del post col player dell'episodio, e visto che avete catalogato il minuto e secondo di inizio delle recensioni, chiacchiere e quant'altro: sarebbe possibile cliccare ad esempio su una recensione ed essere "rimbalzati" a quell'episodio del podcast, nell'istante esatto di inizio della recensione stessa?
immagino che quelli del post dovrebbero lavorare un attimo per supportare la funzionalità (basterebbe passare la info in query string e gestirla lato ilpost, ma ne sapete più di me): magari zampini può chiedere se sono interessati a farlo, anche perchè come funzionalità potrebbe essergli utile in tutti i podcast (già immagino il lonk al podcast di bordone con il minuto/secondo di tutte le sue imitazioni)

**Comment by pdonadeo** [comment]
Ciao Diego, usando un po' il sito (sì, lo faccio, lo uso davvero per cercare le recensioni di giochi proposti da Steam) la tua idea è proprio la prima cosa che uno vorrebbe: la possibilità di saltare direttamente al punto esatto dove si parla di un certo titolo.

Purtroppo allo stato attuale è possibile solo con i vecchi episodi su YouTube. Non so se @zampale vuole provare a chiedere se sia possibile, o meglio: è certamente possibile per Il Post implementare una piccola modifica al player in modo da supportare una notazione stile YouTube per condividere un istante preciso ma non so se hanno voglia e tempo di farlo.

**Reply by zampale** [reply]
Chiedere non mi costa proprio nulla: se mi dite esattamente cosa dovrebbero fare faccio subito (spiegatelo come a un bambino di 3 anni, cioè io).

**Reply by pdonadeo** [reply]
Ecco la spiegazione, da copiare e incollare:

Quando si condivide un video di YouTube c'è la possibilità di includere nel link il momento da cui iniziare la riproduzione, ad esempio: https://youtu.be/W0I0sGI625k?t=309 Qui iniziamo da 5 minuti e 9 secondi (309 secondi).

Sarebbe bello poter fare lo stesso col player utilizzato da Il Post, per esempio nell'episodio 50 di Joypad, minuti 32'01'' si parla di Journey: https://www.ilpost.it/episodes/ep-50-quello-con-lo-spietato-torneo-infernale?t=1921 (1921 ← 32×60 + 1).

**Comment by DGuidi** [comment]
in breve, quello che fa youtube quando condividi un video partendo da uno specifico momento
![image](https://user-images.githubusercontent.com/216627/186677528-a1d4f589-2a9b-4b10-a845-033d82ed7db0.png)

comunque al momento, il risultato del click sul tasto play di un episodio
![image](https://user-images.githubusercontent.com/216627/186676505-8235fa5a-cedb-4614-a659-c39a552d1aa1.png)
è l'apertura dell'episodio stesso sul sito del post all'istante iniziale
![image](https://user-images.githubusercontent.com/216627/186676721-9c33abfc-f4da-42d8-8cdb-46caf48fed5c.png)

un click invece in questa sezione (a caso, yarr!!)
![image](https://user-images.githubusercontent.com/216627/186676875-b1b2644e-8016-419f-aa5e-f8beb62083a2.png)
dovrebbe aprire un indirizzo fatto in questo modo 
https://www.ilpost.it/episodes/ep-52-quello-con-la-gente-che-deve-stare-calma/?t=140

la `?t=140` è un parametro in query string, ovvero un _qualcosa_ che i programmatori del sito web del post dovrebbero gestire in modo che il player del podcast sia inizializzato per riprodurre lo start dal minuto "2:20", e non da "0:00"

di fatto agli autori di questo sito immagino sia una modifica da poco, mentre a "ilpost.it" potrebbe costare di più: però ecco la funzionalità sarebbe condivisibile per tutti i podcast, quindi magari può interessare loro implementarla perchè poi negli articoli/newsletter ecc ecc potrebbero inserire dei link fatti allo stesso modo, per pubblicizzare specifici momenti dei podcast.

@pdonadeo ovviamente correggimo se mi sbaglio o se sono impreciso...

**Reply by MarcoJul** [reply]
Faccio un appunto mentre stiamo sognando questa integrazione (devo dire che mentre la sviluppavo ci pensavo anche io, ma non ho avuto la sinapsi per condividerla😅).

Il click lo farei proprio sul Timestamp (chiedo poi al REPARTO GRAFICO lumi su come si potrebbe integrare un'icona che sia abbastanza chiaro) e lascerei il click sul titolo del gioco per aprire la modale di approfondimento.

Nel frattempo vedo di sistemare i vari bug che ho lasciato in giro😉

**Comment by agofa** [comment]
Idea ottima, tanto io la dovrò solo testare, quindi che mi frega della complessità? :-)
Però considerate che i minuti:secondi indicati non sono ovviamente precisi al microsecondo
Io ho cercato di mettere qualche secondo prima dell'effettivo inizio della citazione
